packages/editor-ui/src/components/RunDataSchema.test.ts

```diff
@@ -51,13 +54,23 @@ const aiTool = createTestNode({
 	disabled: false,
 });
 
+const unknownNodeType = createTestNode({
+	name: 'Unknown Node Type',
+	type: 'unknown',
+});
+
+const defaultNodes = [
+	{ name: 'Manual Trigger', indicies: [], depth: 1 },
+	{ name: 'Set2', indicies: [], depth: 2 },
+];
+
 async function setupStore() {
```

Comment:
Bug somehwere - in the empty item case
<img width="1470" alt="Screenshot 2024-11-18 at 11 04 38" src="https://github.com/user-attachments/assets/f183bfdd-34ce-476d-a965-dd126655c54f">